tdrs-frontend/src/assets/uswds/_uswds-theme-utilities.scss

```diff
@@ -31,7 +31,7 @@ used by utilities or layout grid
 */
 
 $theme-utility-breakpoints: (
-  "card": false,
+  "card": true,
```

Comment:
what's this change about? @jorgegonzalez

Reply:
This turns on breakpoints for "card-sized" viewports (i.e. width of at least 160px).

See the `card` utility class is referenced in the Submit Files Button:

https://github.com/raft-tech/TANF-app/blob/0293259eadb6036348f636ac443aa0f7b69c59f6/tdrs-frontend/src/components/UploadReport/UploadReport.jsx#L43

For the smallest viewports, we need that class defined to provide additional margin to the button:

Before:
![Screen Shot 2021-03-18 at 3 24 22 PM](https://user-images.githubusercontent.com/12901172/111685775-b26f1780-87fe-11eb-8a14-b323748dbdcf.png)

After:
![Screen Shot 2021-03-18 at 3 24 50 PM](https://user-images.githubusercontent.com/12901172/111685770-b13dea80-87fe-11eb-8c37-5e39e1d45f24.png)